r2/r2/templates/link.html

```diff
@@ -134,18 +135,17 @@
 </%def>
 
 <%def name="summary()">
-  <% plain_text = killhtml(renderpolls(thing._summary(), thing)) %>
-  <% first_bit = first_words(plain_text, 50) %>
+  <% article_summary = ArticleSummary(thing) %>
 
-  ${unsafe(first_bit)}
-  % if thing._has_more():
+  ${unsafe(article_summary.first_bit())}
+  % if article_summary.has_more():
```

Comment:
This branch of the if statement is untested. I'm not quite sure how to trigger it. I don't see the point of it, given the `elif` branch below.